### ISSUE TITLE
[Feature Request] Set Default Home Layout Options for Created Accounts 

### ISSUE THREAD

**Issue by DER31K** [hrfee/jellyfin-accounts#17]
Thanks for building this awesome project. I'm so close to automating my on boarding process so that I can invite my family/friends to my media sever with minimal effort on my part (and theirs) 

One hurdle I've been trying to overcome is to be able to automatically set the layout that new users are presented with.

I think most of my users want library menu to be simple, and the way Jellyfin displays the dashboard by default does not work well.

Is it possible to build a feature where we can copy the Settings > Home options from one user and use them to create new users?

**Comment by hrfee** [comment]
Emby's API documentation (It's roughly the same for jellyfin) is a bit convoluted, but it says to do this by calling '/Users/{Id}/Configuration' 'Requires authentication as user'. It says this for changing a user's password too, but if you include it when you create a user, it works fine (took me sooo long to realise that when i wrote the api client). I'll give it a go now, hopefully it works. 
I see you've forked the repository as well, are you working on this yourself? if so, embys [swagger](http://swagger.emby.media/?staticview=true#/) is a good resource.

**Comment by hrfee** [comment]
Ignore the previous comment, i've only just noticed that you can edit any user's preferences in the ui, so this is definitely possible.

**Comment by hrfee** [comment]
Right now i've got it to set the Library order, but it seems most of the other settings are grabbed as separate requests so it might take a while to get everything done.

**Comment by hrfee** [comment]
I've just added this in 674636b3a236b3556d1ee52f8f4abca8a4f4c6d9
You can set the default policy and homescreen layout via `jf-accounts --get_defaults`.

**Comment by hrfee** [comment]
Also, 39a27eb7629f45f1dd720b61b1b1f910eb7eb1d3 now allows you to do this through the web ui instead of the command line.

**Comment by DER31K** [comment]
Honestly I can't thank you enough for implementing this, especially with the quick turnaround - totally unexpected! Huge thanks! 

With this addition, I think I'll be able to let my friends get their accounts up and running with minimal input from me.

Will test it out tonight.

BTW I pulled your code into a private repo where I was overriding some of the auth/invite code checks, (definitely not recommended for most users). 

I did this so that I can use the backend python you've written as an API.  (I have some very crude/old front end web development skills from a past life... (html,css, Javascript), I've designed a secure, branded web UI that makes Javascript calls to your backend for account creation/invites.  I prefer my custom UI because the look and feel is more consistent with what my users would expect on my server.
